pages/search.vue

```diff
@@ -555,16 +557,23 @@ export default {
         this.orFacets.splice(index, 1)
       } else {
         if (elementName === 'categories:purpur') {
-          this.orFacets.push('categories:paper')
-          this.orFacets.push('categories:spigot')
-          this.orFacets.push('categories:bukkit')
+          if (this.orFacets.includes('categories:paper'))
+            this.orFacets.push('categories:paper')
+          if (this.orFacets.includes('categories:spigot'))
+            this.orFacets.push('categories:spigot')
+          if (this.orFacets.includes('categories:bukkit'))
+            this.orFacets.push('categories:bukkit')
         } else if (elementName === 'categories:paper') {
-          this.orFacets.push('categories:spigot')
-          this.orFacets.push('categories:bukkit')
+          if (this.orFacets.includes('categories:spigot'))
+            this.orFacets.push('categories:spigot')
+          if (this.orFacets.includes('categories:bukkit'))
+            this.orFacets.push('categories:bukkit')
         } else if (elementName === 'categories:spigot') {
-          this.orFacets.push('categories:bukkit')
+          if (this.orFacets.includes('categories:bukkit'))
+            this.orFacets.push('categories:bukkit')
         } else if (elementName === 'categories:waterfall') {
-          this.orFacets.push('categories:bungeecord')
+          if (this.orFacets.includes('categories:bungeecord'))
+            this.orFacets.push('categories:bungeecord')
```

Comment:
Isn't the logic on most of these if statements reversed? i.e., this line 576 is only adding the bungeecord orFacet if it already includes the bungeecord orFacet. It should be doing this if it does *not* already include that orFacet.